.jsdoc.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
     copyright: 'Copyright 2020 Google LLC',
     includeDate: false,
     sourceFiles: false,
-    systemName: '@google-cloud/documentai',
```

Comment:
I think we should fix this upstream in the configuration, with the `package_name` hint.